src/main/target/common_fc_pre.h

```diff
@@ -124,6 +124,7 @@
 #define VTX_CONTROL
 #define VTX_SMARTAUDIO
 #define VTX_TRAMP
+#define CAMERA_CONTROL
```

Comment:
I'd like to see all build flag have a `USE_` prefix. So it would be good to see it for the `VTX_` items, but, yes, it should be in a separate PR.